charts/konk/templates/tests/test-connection.yaml

```diff
@@ -8,8 +8,27 @@ metadata:
     "helm.sh/hook": test-success
 spec:
   containers:
-    - name: wget
-      image: busybox
-      command: ['wget']
-      args: ['{{ include "konk.fullname" . }}:{{ .Values.service.port }}']
+  - name: kind
+    image: "{{ .Values.kind.image.repository }}:{{ .Values.kind.image.tag | default .Chart.AppVersion }}"
+    imagePullPolicy: {{ .Values.kind.image.pullPolicy }}
+    command:
+      - kubectl
+    args:
+      - --server=https://{{ include "konk.fullname" . }}:{{ .Values.service.port }}
```

Comment:
Todo default kubeconfig to fqdn service name

Reply:
Completed in https://github.com/infobloxopen/konk/pull/16